core/modules/field/field.module

```diff
@@ -1118,6 +1118,49 @@ function field_get_items($entity_type, $entity, $field_name, $langcode = NULL) {
 }
 
 /**
+ * Returns field values in the language they currently would be displayed.
+ *
+ * Fields with a 'value' key are supported, such as text and number fields. 
+ * Fields which do not have a 'value' key such as terms (which store term IDs
+ * with a 'tid' key), or files (an 'fid' key), will return an empty array.
+ *
+ * @param $entity_type
+ *   The type of $entity; e.g., 'node' or 'user'.
```

Comment:
We don't need this parameter. You can now always just call $entity->entityType().